verification/curator-service/api/src/model/upload.ts

```diff
@@ -0,0 +1,28 @@
+import { UploadSummaryDocument, uploadSummarySchema } from './upload-summary';
+
+import mongoose from 'mongoose';
+
+const uploadSchema = new mongoose.Schema({
+    status: {
+        type: String,
+        required: true,
+    },
+    summary: {
+        type: uploadSummarySchema,
+        required: true,
+    },
+    created: {
+        type: Date,
+        default: Date.now(),
```

Comment:
should this be `Date.now` instead of `Date.now()` ? (aka. callback vs raw date value)

Reply:
Yep, typo on my part, good catch.